### ISSUE TITLE
A page showing all active video processes & meta data?

### ISSUE THREAD

**Issue by marcoherbst** [evercam/evercam-admin#257]

**Comment by ijunaid8088** [comment]
is this valid? was it about Snapshot Extractor?

**Comment by marcoherbst** [comment]
Ideally, any ffmpeg process ..... but, initially, live streaming (rtsp > rtmp / HLS) only.

**Comment by marcoherbst** [comment]
how much stream meta data you can show me.
Separate Stream IN from Stream Out  ( I know today they are the same , but soon they will be different)

**Comment by marcoherbst** [comment]
Azhar also mentioned: # viewers & IPs of viewers.

**Comment by azharmalik3** [comment]
@marcoherbst I have created report for [active video sessions](https://admin.evercam.io/meta_datas). This report will show all active video sessions for `RTMP` and `HLS`. Tried to add username in the list but it needs little bit extra work to add username so I leave empty.

Also, RTMP always gives server local IP because RTMP server sends request locally whenever stream play. Here is code reference https://github.com/evercam/evercam-devops/blob/master/ansible/nginx/templates/nginx.conf.j2#L109
